src/test/java/com/puppycrawl/tools/checkstyle/BaseCheckTestSupport.java

```diff
@@ -58,6 +62,22 @@ protected Checker createChecker(Configuration checkConfig)
         return checker;
     }
 
+    protected Checker createChecker(Configuration checkConfig, boolean printSeverity)
+        throws Exception {
+
+        final DefaultConfiguration dc = createCheckerConfig(checkConfig);
+        final Checker c = new Checker();
```

Comment:
TC says: `Variable named c doesn't have type 'char' or 'java.lang.Character'`

Just rename it to `checker`.